libs/partners/mistralai/langchain_mistralai/chat_models.py

```diff
@@ -89,16 +92,19 @@ def _create_retry_decorator(
 
 def _convert_mistral_chat_message_to_message(
     _message: MistralChatMessage,
+    stop: Optional[List[str]] = None,
 ) -> BaseMessage:
     role = _message.role
     if role == "user":
-        return HumanMessage(content=_message.content)
+        return HumanMessage(content=_enforce_stop_tokens(_message.content, stop))
```

Comment:
I don't think we should be dropping the stop tokens form the content.  The stop token cannot be recovered by the client if the stop token was a list of stop tokens. Recovery of the token is needed for parsing structured content.

If I understand correctly, there's no way to recover this information easily via `.invoke` and `.stream` etc